harness/tests/experiment/pytorch/test_deepspeed_autotuning.py

```diff
@@ -47,11 +47,11 @@
 
 
 MODEL_INFO_PROFILE_METRIC_FIXTURE = {
-    "num_params": 60192808,
-    "trainable_num_params": 60192808,
-    "activation_mem_per_gpu": 89828352,
+    "num_params": 1,
+    "trainable_num_params": 1,
+    "activation_mem_per_gpu": 1,
     "rank": 0,
-    "gpu_mem": 15843721216,
+    "gpu_mem": 32,
 }
```

Comment:
Why would we change this? wouldn't we want values that are reasonably realistic?

Reply:
It made binary search tests run for an annoyingly long time. Like ~300 different checks per stage. These settings dropped that down to ~30.  Can change it back, if you prefer, though.

Reply:
I suppose it's fine. I just want to make sure our tests are reasonably representative